### PR TITLE
Support completion for kubectl apply view/edit-last-applied

### PR DESCRIPTION
**Pull request by superbrothers**
**What this PR does / why we need it**: With this PR, `kubectl apply view/edit-last-applied` support completion.
```
$ kubectl apply view-last-applied <tab>
certificatesigningrequest  configmap           daemonset   event                    job            node                   pod                  podtemplate            rolebinding  serviceaccount  storageclass
clusterrolebinding         controllerrevision  deployment  horizontalpodautoscaler  namespace      persistentvolume       poddisruptionbudget  replicaset             secret       statefulset     
componentstatus            cronjob             endpoints   ingress                  networkpolicy  persistentvolumeclaim  podsecuritypolicy    replicationcontroller  service      status
$ kubectl apply view-last-applied deployment nginx<tab>
nginx   nginx2
```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
`kubectl apply view/edit-last-applied support completion.
```

### CODE REVIEW

File: pkg/kubectl/cmd/cmd.go [hunk 1]
```diff
@@ -168,7 +168,8 @@ __kubectl_require_pod_and_container()
 __custom_func() {
     case ${last_command} in
         kubectl_get | kubectl_describe | kubectl_delete | kubectl_label | kubectl_edit | kubectl_patch |\
-        kubectl_annotate | kubectl_expose | kubectl_scale | kubectl_autoscale | kubectl_taint | kubectl_rollout_*)
+        kubectl_annotate | kubectl_expose | kubectl_scale | kubectl_autoscale | kubectl_taint | kubectl_rollout_* |\
+        kubectl_apply_edit-last-applied | kubectl_apply_view-last-applied)
```
Comment (shiywang) [inline pkg/kubectl/cmd/cmd.go]: I think there's also a `set-last-applied` we add it.

**Comment by shiywang** [comment]
@mengqiy @superbrothers I tried on my laptop, it works well, there's only one `set-last-applied` left then we are good to go.
```
➜  amd64 git:(last-applied) ./kubectl apply view-last-applied
certificatesigningrequest  persistentvolume         
clusterrolebinding         persistentvolumeclaim    
componentstatus            pod                      
configmap                  poddisruptionbudget      
controllerrevision         podsecuritypolicy        
cronjob                    podtemplate              
daemonset                  replicaset               
deployment                 replicationcontroller    
endpoints                  rolebinding              
event                      secret                   
horizontalpodautoscaler    service                  
ingress                    serviceaccount           
job                        statefulset              
namespace                  status                   
networkpolicy              storageclass             
node                                                
```

**Comment by superbrothers** [comment]
@shiywang `kubectl apply set-last-applied` don't have `RESOURCE/NAME` argument unlike `view/edit-last-applid`. `set-last-applied` already supports completion.

> kubectl apply set-last-applied -f FILENAME [options]
> kubectl apply view-last-applied (TYPE [NAME | -l label] | TYPE/NAME | -f FILENAME) [options]
> kubectl apply edit-last-applied (RESOURCE/NAME | -f FILENAME) [options]

**Comment by shiywang** [comment]
@superbrothers oh, sorry : ) 
/lgtm

**Comment by superbrothers** [comment]
@shiywang Thank you for your review! ;)

**Pull request merged**